mwapi/session.py

```diff
@@ -39,15 +41,21 @@ def __init__(self, host, api_path="/w/api.php", session=None):
         self.api_path = api_path
         self.api_url = host + api_path
         self.session = session or requests.Session()
-        self.is_authenticated = False
-        """
-        `bool` : Is the session sending authenticated requests
-        """
 
     def _request(self, method, params=None, data=None, files=None):
         resp = self.session.request(method, self.api_url, params=params,
                                     data=data, files=files, stream=True)
-        return resp.json()
+
+        try:
+            doc = resp.json()
```

Comment:
doc feels like a confusing variable name here - I usually use a variant of 'result', or spell out document fully.